src/plugins/intel_gna/gna_graph_compiler.cpp

```diff
@@ -264,17 +264,18 @@ void GNAGraphCompiler::ConvolutionPrimitive(InferenceEngine::CNNLayerPtr layer)
         std::swap(convolution._padding_x, convolution._padding_y);
         std::swap(convolution._stride_x, convolution._stride_y);
         std::swap(convolution._dilation_x, convolution._dilation_y);
-    }
-
-    // Map 2d convolution to 1d if it's possible
-    if (GNAConvolutionLayer::isMappableFrom2DTo1D(in_height, in_width, convolution._kernel_x, convolution._stride_x)) {
+    } else if (GNAConvolutionLayer::isMappableFrom2DTo1D(in_height, in_width, convolution._kernel_x, convolution._stride_x)) {
+        // Map 2d convolution to 1d if it's possible
+        convolution._stride_x = (convolution._stride_y * in_width);
```

Comment:
This PR has conflicts with https://github.com/openvinotoolkit/openvino/pull/9474 to support new format of kaldi IRs. Let's start with my PR to unlock https://github.com/openvinotoolkit/openvino/pull/5744

Reply:
Conflicting part is not significant, I could revert some part from my PR, or I can help you to resolve these conflicts from yours.

Reply:
@nkogteva  conflicting part was removed

Reply:
@nkogteva could you check if it is OK for you?

Reply:
@kbruniec I think we need to run AC validation for this PR

Reply:
@kbruniec I think we need to run AC validation for this PR

Reply:
@nkogteva It may be worth to do so. But please note this snippet is outdated.

Reply:
Accuracy results looks ok